portal/eproms/templates/eproms/404.html

```diff
@@ -37,7 +37,7 @@ <h4>
                     <a href="/" class="btn feature-btn feature-btn-primary">{{ _("Back To Home") }}</a>
                 </div>
                 <div class="tnth-splash-bottom">
-                    <a href="/explore" class="tnth-splash-bottom-link"><span class="hidden-xs">{{ _("Explore How This Works") }}</span><span class="hidden-sm hidden-md hidden-lg">{{ _("Explore") }}</span></a><span class="tnth-splash-bottom-link separator"></span><a href="/about" class="tnth-splash-bottom-link"><span class="hidden-xs">{{ _("Learn About TrueNTH") }}</span><span class="hidden-sm hidden-md hidden-lg">{{ _("Learn") }}</span></a>
+                    <a href="/about" class="tnth-splash-bottom-link"><span class="hidden-xs">{{ _("Learn About TrueNTH") }}</span><span class="hidden-sm hidden-md hidden-lg">{{ _("Learn") }}</span></a>
```

Comment:
I am not sure if `Learn About Truenth` has been translated previously. We could use the same verbiage `Explore How This Works` to make sure no translation issue.

Reply:
I didn't add that - it was already there.  I expect it is already translated, that's all old data, right @achen2401 ?

Reply:
ah, thank you, paul. sorry I missed that.